src/GitVersionCore.Tests/IntegrationTests/MasterScenarios.cs

```diff
@@ -24,10 +24,12 @@ public void CanHandleContinuousDelivery()
                     }
                 }
             };
-            using var fixture = new EmptyRepositoryFixture();
-            fixture.Repository.MakeATaggedCommit("1.0.0");
-            fixture.Repository.MakeCommits(2);
-            fixture.AssertFullSemver(config, "1.0.1+2");
+            using (var fixture = new EmptyRepositoryFixture())
```

Comment:
Could you please revert to the new syntax (C# 8 `using`)? We need to keep the same format and syntax in the entire code base

Reply:
Okei, I guess I also should do that for other testfile I was working with.

Reply:
Yes